src/orders-update.js

```diff
@@ -64,38 +64,51 @@ export default class OrdersUpdate {
       })
   }
 
-  // Get the state ID from the API based on the key
-  // getStateReference :: String -> Promise -> String
-  getStateReference (key) {
+  // Get the ID from the API based on the key
+  // getStateReference :: (String, String, String) -> Promise -> String
+  getIdFromKey (key, typeId, endpoint) {
+    if (typeof key === 'object')
```

Comment:
Could be done with a wrapper function or something like that, *but* let's not think about that too much since the code doesn't make sense in the first place. We never want to return an object.